BackBLAST

```diff
@@ -186,10 +190,22 @@ function perform_setup() {
   echo "[ $(date -u) ]: Command run: ${SCRIPT_NAME} setup ${original_arguments}" >&2
 
   # Make the template files for the run
-  make_run_templates ${template_config} ${query_filepath} ${query_genome_filepath} ${subject_genome_directory} \
-    ${genome_extension} ${output_directory} ${threads} ${phylogenetic_tree_newick} ${bootstrap_cutoff} ${root_name} ${evalue} ${pident}
+  ${utils_dir}/generate_run_templates.sh \
```

Comment:
Rather than sourcing this script and calling functions from it (as done in the old code), I now just call the script itself.